hudi-io/src/test/java/org/apache/hudi/io/hfile/TestHFileReader.java

```diff
@@ -268,6 +271,33 @@ public void testReadHFilePointAndPrefixLookup(String filename,
     verifyHFileRead(filename, numEntries, KEY_CREATOR, VALUE_CREATOR, keyLookUpInfoList);
   }
 
+  @Test
+  public void testReadHFileCorrupted() throws Exception {
```

Comment:
Do we need this unit test ? We can move the validation of the HFile as an assertion in `testSeekKeyDoesNotExist` or remove it completely ?